app/helpers/proxmox_vm_volumes_helper.rb

```diff
@@ -52,12 +52,14 @@ def parsed_typed_volumes(args, type, parsed_vm)
   end
 
   def parse_hard_disk_volume(args)
+    logger.debug(format(_('parse_hard_disk_volume(): args=%<args>s'), args: args))
     disk = {}
     disk[:id] = args['id'] if args.key?('id')
     disk[:volid] = args['volid'] if args.key?('volid')
     disk[:storage] = args['storage'].to_s if args.key?('storage')
     disk[:size] = args['size'].to_i if args.key?('size')
-    add_disk_options(disk, args)
+    add_disk_options(disk, args) unless args.key?('options')
+    disk[:options] = args['options'] if args.key?('options')
```

Comment:
I'd make this a traditional `if`/`else`
```suggestion
    if args.key?('options')
      disk[:options] = args['options']
    else
      add_disk_options(disk, args)
    end
```